hubblestack/files/hubblestack_nova/misc.py

```diff
@@ -511,15 +506,17 @@ def check_core_dumps(reason=''):
     """
     Ensure core dumps are restricted
     """
-    hard_core_dump_value = _execute_shell_command("grep -R -E \"hard +core\" /etc/security/limits.conf /etc/security/limits.d/ | awk '{print $4}'", python_shell=True).strip()
-    hard_core_dump_value = hard_core_dump_value.split('\n') if hard_core_dump_value != "" else []
-    if '0' in hard_core_dump_value:
-        return True
-
-    if hard_core_dump_value is None or hard_core_dump_value == [] or hard_core_dump_value == "":
-        return "'hard core' not found in any file"
-
-    return str(hard_core_dump_value)
+    # TODO fix/replace this broken code - maybe with some LIBC library
+    # hard_core_dump_value = _execute_shell_command("grep -R -E \"hard +core\" /etc/security/limits.conf /etc/security/limits.d/ | awk '{print $4}'", python_shell=True).strip()
+    # hard_core_dump_value = hard_core_dump_value.split('\n') if hard_core_dump_value != "" else []
+    # if '0' in hard_core_dump_value:
+    #     return True
+    #
+    # if hard_core_dump_value is None or hard_core_dump_value == [] or hard_core_dump_value == "":
+    #     return "'hard core' not found in any file"
+    #
+    # return str(hard_core_dump_value)
+    return True
```

Comment:
Why is this just being commented out and replaced with `return True`?

Reply:
the check the code runs is successful even for documentation lines which are commented out, a little more research is needed to fix it but in the interim I commented it out so it stops returning incorrect results